contracts/token/Token.sol

```diff
@@ -1,27 +1,40 @@
-pragma solidity ^0.5.10;
+pragma solidity ^0.6.0;
 
 import "./MintableAndBurnable.sol";
 import "./IToken.sol";
 
 contract Token is IToken, MintableAndBurnable {
-    string public name = "TREXDINO";
-    string public symbol = "TREX";
-    string public version = "1.2";
-    uint8 public decimals = 0;
-    address public onchainID = 0x0000000000000000000000000000000000000000;
+    string private tokenName = "TREXDINO";
+    string private tokenSymbol = "TREX";
+    string private tokenVersion = "1.2";
+    uint8 public override decimals = 0;
+    address public override onchainID = 0x0000000000000000000000000000000000000000;
 
     constructor(address _identityRegistry, address _compliance) public TransferManager(_identityRegistry, _compliance) {}
 
+    function name() public view override returns (string memory) {
```

Comment:
i don't see the point of these getters, why not keeping the variables public ?

Reply:
Keeping variables public will make the return types different from getter functions and thus will not allow it to override the IToken state variables